doc/source/examples/protoselect_adult_cifar10.ipynb

```diff
@@ -0,0 +1,1140 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    # search for the best epslion-radius value
</pre><p>"epslion" -&gt; "epsilon"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/626/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>